docs/sources/developers/plugins/metadata.md

```diff
@@ -22,25 +22,24 @@ The plugin.json file is required for all plugins. When Grafana starts, it scans
-| `logs`               | boolean                       | No       | For data source plugins, if the plugin supports logs.                                                                                                                                                                                                                                                                                                                                                   |
-| `metrics`            | boolean                       | No       | For data source plugins, if the plugin supports metric queries. Used in Explore.                                                                                                                                                                                                                                                                                                                        |
-| `preload`            | boolean                       | No       | Initialize plugin on startup. By default, the plugin initializes on first use.                                                                                                                                                                                                                                                                                                                          |
+| `logs`               | boolean                       | No       | For data source plugins, if the plugin supports logs. Not used in Grafana core but may be used by external plugins.                                                                                                                                                                                                                                                                                     |
+| `metrics`            | boolean                       | No       | For data source plugins, if the plugin supports metric queries. Used to enable the plugin in the panel editor.                                                                                                                                                                                                                                                                                          |
+| `pascalName`         | string                        | No       | The PascalCase name for the plugin. Used for creating machine-friendly identifiers, typically in code generation. If not provided, defaults to name, but title-cased and sanitized (only alphabetical characters allowed).                                                                                                                                                                              |
```

Comment:
Will reader know what PascalCase is?

Reply:
actually, since we are marking some properties as "internal only" and remove it from the docs, this is one of those cases.